frontend/src/routes.tsx

```diff
@@ -4,11 +4,13 @@ import Register from "./pages/Register";
 import StartupView from "./pages/StartupView";
 import InvestorDashboard from "./pages/InvestorDashboard";
 import Inbox from "./pages/Inbox";
+import RegisterStartup from "./pages/RegisterStartup";
 
 export const routes = [
   { path: "/", element: <Home /> },
   { path: "/login", element: <Login /> },
   { path: "/register", element: <Register /> },
+  { path: "/registerstartup", element: <RegisterStartup /> },
```

Comment:
The new route path "/registerstartup" is camel-cased; prefer a consistent URL style (kebab-case) like "/register-startup" for readability and SEO. Rename the route path to use kebab-case.
```
{ path: "/register-startup", element: <RegisterStartup /> },
```